x/stakeibc/abci.go

```diff
@@ -14,4 +15,12 @@ import (
 // BeginBlocker of stakeibc module
 func BeginBlocker(ctx sdk.Context, k keeper.Keeper, bk types.BankKeeper, ak types.AccountKeeper) {
 	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyBeginBlocker)
+
+	// Iterate over all host zones and verify redemption rate
+	for _, hz := range k.GetAllHostZone(ctx) {
+		rrSafe, _ := k.IsRedemptionRateWithinSafetyBounds(ctx, hz)
```

Comment:
can we populate the err and flow it into the log below? will help debug in case this ever triggers